css/tabBar.css

```diff
@@ -230,14 +228,14 @@ body.dark-mode #overlay {
 
 .tab-item .tab-icon.permission-request-icon {
 	border: 1px rgba(0, 0, 0, 0.3) solid;
-    padding: 0.16em;
-    margin-right: 0.66em;
-    margin-top: -0.2em;
-    border-radius: 0.75em;
+	padding: 0.15em 0.17em;
+	margin-right: 0.66em;
+	margin-top: -0.2em;
+	margin-left: -0.3em;
+	border-radius: 0.75em;
 }
-.tab-item .tab-icon.permission-request-icon i {
-	margin: 0 0.15em;
-	vertical-align: bottom;
+.tab-item .tab-icon.permission-request-icon .i {
+	vertical-align: middle;
```

Comment:
This button is probably a little too large now, or it needs to be moved upwards:

<img width="258" alt="Screen Shot 2020-07-16 at 11 25 55 AM" src="https://user-images.githubusercontent.com/10314059/87696969-2c726c80-c757-11ea-9f68-40a3af31a534.png">

(It looks fine in your original screenshot, I guess it regressed at some point)

Reply:
![image](https://user-images.githubusercontent.com/11247099/88450415-50cdf780-ce81-11ea-9bfe-f8708a9f4bf5.png)

Changed to this, is that looks good to you?

Reply:
BTW is there any website that I can use to test mic and camera at the same time?

Reply:
Yeah, it looks fine.

Google meet will make a single request for camera and mic, which results in this:
<img width="127" alt="Screen Shot 2020-07-25 at 5 00 49 PM" src="https://user-images.githubusercontent.com/10314059/88467065-892d0000-ce98-11ea-99be-73c9ac557b09.png">

I think it's possible to have separate requests for camera and microphone (which would create two separate bubbles), but I haven't found a way to do it yet.

You can also do this:

```
window.addEventListener("click", function() {
navigator.mediaDevices.getUserMedia({video: true, audio: true})

Notification.requestPermission();

});
```

<img width="209" alt="Screen Shot 2020-07-25 at 5 06 21 PM" src="https://user-images.githubusercontent.com/10314059/88467125-3142c900-ce99-11ea-8ead-ec82154d9012.png">